pcl-odk.yml

```diff
@@ -25,4 +25,4 @@ import_group:
     - id: bfo
     - id: cl
     - id: omo
-robot_java_args: '-Xmx8G'
\ No newline at end of file
+robot_java_args: '-Xmx24G'
```

Comment:
Wrong file. src/ontology/pcl-odk.yml. Delete this one.

Also: don't forget update_repo

Reply:
24 GB is _way_ too much. 8 is lower limit, try 12.